sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -0,0 +1,72 @@
+import { IsNotEmpty, IsOptional } from "class-validator";
+
+export class CreateInstitutionDto {
+  @IsNotEmpty()
+  legalOperatingName: string;
+
+  @IsOptional()
+  operatingName: string;
+
+  @IsNotEmpty()
+  primaryPhone: string;
+
+  @IsNotEmpty()
+  primaryEmail: string;
+
+  @IsNotEmpty()
+  website: string;
+
+  @IsNotEmpty()
+  regulatingBody: string;
+
+  @IsNotEmpty()
+  establishedDate: string;
```

Comment:
In the future, we need to review the validations that UI is doing and API is not.
This field is not supposed to be a date in the future, for instance.
Don't need to change it right now.

Reply:
Didnt really get this. It is a string anyways in the dto. Its a date in the model.

Reply:
The requirement says "Established Date (may not be a future date)".
It doesn't matter if the API is accepting a string of data. From a business perspective, it is a date and it should not be in the future. The API should validate this and any other similar validation. We should never rely only on the UI to execute the validations, right?
Please note that I did not request this to be changed right now because we have other similar issues like this, for instance, the SIN validation that is happening only in the UI.

Reply:
Holding it for now.